install.py

```diff
@@ -14,26 +13,31 @@
 import datetime
 
 
-tmp_bench_repo = os.path.join('/', 'tmp', '.bench')
-tmp_log_folder = os.path.join('/', 'tmp', 'logs')
+tmp_bench_repo = os.path.join("/", "tmp", ".bench")
+tmp_log_folder = os.path.join("/", "tmp", "logs")
```

Comment:
Not sure what changed in this file, can you undo formatting changes?

Reply:
pre-commit changed ' to "

Reply:
So we have precommit without existing code already compliant? 🤔 

Still best to undo this and do it in one separate commit.